wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -38,10 +38,10 @@ The NAT keep tabs on potential NAT candidates over long periods of time, and occ
 
 ## Team members
 
-The [Nomination Assessment Team group page](https://osu.ppy.sh/groups/7) lists all of the team members.
-
 *Note: All NAT members speak English in addition to the language(s) listed below unless noted otherwise.*
 
+The [Nomination Assessment Team group page](https://osu.ppy.sh/groups/7) lists all of the team members. In addition to areas mentioned below, all NAT members take part in [reviewing visual and aural content included in beatmaps](/wiki/Rules/Visual_Content_Considerations#getting-your-image-assessed).
```

Comment:
```suggestion
The [Nomination Assessment Team group page](https://osu.ppy.sh/groups/7) lists all of the team members. In addition to areas mentioned below, all NAT members take part in [reviewing audio and visual content included in beatmaps](/wiki/Rules/Visual_Content_Considerations#getting-your-image-assessed).
```

aural is technically correct but almost never used outside of scientific papers from my experience